### DISCUSSION TITLE
Get maximum upload attachment size from client through API

### DISCUSSION THREAD

**Discussion by jackvt93** [mastodon/mastodon#27955] [Q&A] [answered]
Is there any way to get attachment maximum size configuration from client?. I see that each instance have different upload allowed maximum size and the client needs to know the file size that allowed to upload for checking or warn users to choose back before sending to server.

Thanks

**Comment by deepchhatralia** [comment, marked as answer]
You can try calling the below instance API which provides instance details along with the media limits
https://<SERVER_NAME>/api/v2/instance

Ex: https://mastodon.social/api/v2/instance
